internal/ais/combinemd.go

```diff
@@ -0,0 +1,138 @@
+package ais
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/antchfx/xmlquery"
+
+	"github.com/artefactual-sdps/preprocessing-sfa/internal/fsutil"
+)
+
+const CombineMDActivityName = "combine-metadata-files"
+
+type (
+	CombineMDActivity       struct{}
+	CombineMDActivityParams struct {
+		AreldaRelPath string
+		METSRelPath   string
```

Comment:
I'm not clear if you are saying "we could pass the absolute paths, if we want to" or "we should be using absolute paths here".

Reply:
Using absolute paths makes the tests a bit harder, if nothing else. It also duplicates the path information in the "WorkingDir" parameter.

Reply:
I think it's better to use absolute paths since we already have them in the workflow (`Destination` params used in the fetch activities), it seems unnecessary to me to recreate them in the activity.

Reply:
Okay, I'll switch to absolute paths.